test/cli/autogen-autoloader/example1.rb

```diff
@@ -20,10 +20,6 @@ class Jazz < Quuz
       class JazBaz
         p 'x'
         require 'in_class'
-
-        def honk
-          require 'in_method'
-        end
```

Comment:
Why are we deleting code from test files?

(Usually deleting code from tests implies that the change causes tests to fail, and we're papering over it by changing the test instead of fixing the code.)

Reply:
I think this was accidental, sorry